cmd/admin-handlers-users.go

```diff
@@ -1719,6 +1719,12 @@ func (a adminAPIHandlers) AddCannedPolicy(w http.ResponseWriter, r *http.Request
 	}
 	setReqInfoPolicyName(ctx, policyName)
 
+	// Reject policy names with commas.
+	if strings.Contains(policyName, ",") {
```

Comment:
Is this the only thing we are restricting?

Reply:
On windows a : won't work either.

Reply:
At least for this PR. Commas cause issues because policies are comma delimited when attached to users/groups. There are other restrictions already in place but this one had not been added yet

Reply:
Is the `:` restriction on windows due to file naming?

Reply:
👍🏼